.github/workflows/build.yml

```
@@ -0,0 +1,65 @@
+name: Build VS Code extension
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+
+permissions:
+  # Write permissions needed for publishing the release
+  contents: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build_vscode_ext:
+    name: Build VS Code extension
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Action is showing the warning
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```
```suggestion
      - name: Checkout
        uses: actions/checkout@v4
```